src/operators.md

```diff
@@ -27,18 +27,20 @@ Infix operators can also be declared to be commutative or associative (or neithe
 The syntax to define a prefix operator looks like this:
 
 ```
-prefix <operatorSymbol> <precedenceNumber> functionName
```

Comment:
I can't leave a comment on line 5, but can we reference  the table where all the operators are defined?

Reply:
I think so, let me figure out how to do it.